backend/lib/app.js

```diff
@@ -32,6 +32,8 @@ if (gitHubRepoUrl) {
   imgSrc.push(url.origin)
   url.hostname = 'avatars.' + url.hostname
   imgSrc.push(url.origin)
+  url.hostname = 'media.' + url.hostname
```

Comment:
this will lead to the hostname `media.avatars.xyz`

instead, do something like
```js
  const hostname = url.hostname
  url.hostname = 'avatars.' + hostname
  imgSrc.push(url.origin)
  url.hostname = 'media.' + hostname
  imgSrc.push(url.origin)
```

Reply:
Great catch 👍 
Updated it

Reply:
```node
> gitHubRepoUrl="https://github.enterprise/org/repo"
'https://github.enterprise/org/repo'
> imgSrc=[]
[]
> const url = new URL(gitHubRepoUrl)
undefined
> const gitHubHostname = url.hostname
undefined
> url.hostname = 'avatars.' + gitHubHostname
'avatars.github.enterprise'
> imgSrc.push(url.origin)
1
> url.hostname = 'media.' + gitHubHostname
'media.github.enterprise'
> imgSrc.push(url.origin)
2
> console.log(imgSrc)
[
  'https://avatars.github.enterprise',
  'https://media.github.enterprise'
]
undefined
```